### ISSUE TITLE
Add 'Unapprove' feature to the Tokens page

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#682]
originally reported in https://github.com/cowprotocol/cowswap/pull/677#issuecomment-1152522325 (case 4)

Currently, we can see all tokens with Approved/Unapproved status on the Tokens page. 
Once a token is approved, it would be nice to have an 'Unapprove' feature for it as well
![image.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/75ad2f0d-53b9-43e2-be35-aafaca9b31b9)
